esp/services/ws_fs/ws_fsService.cpp

```diff
@@ -2237,7 +2237,7 @@ void CFileSprayEx::getDropZoneInfoByIP(double clientVersion, const char* ip, con
         destFile.set(destFileIn);
         Owned<IConstDropZoneInfo> dropZone = constEnv->getDropZoneByAddressPath(ip, destFile.str());
         if (!dropZone)
-            throw MakeStringException(ECLWATCH_DROP_ZONE_NOT_FOUND, "Dropzone not found for network address %s.", ip);
+            throw MakeStringException(ECLWATCH_DROP_ZONE_NOT_FOUND, "Drop zone to target '%s' not found for network address '%s'. Check your system drop zone configuration.", destFile.str(), ip);
```

Comment:
@AttilaVamos
"No dropzone configured for 'IP' and 'PATH'. Check your system dropzone configuration" seems more direct and to the point. @JamesDeFabia  what do you think?
Also, is it "Dropzone" or "drop zone" ?